src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -192,6 +195,7 @@ export class TransactionHelper implements OnModuleInit {
       discountCodes,
       true,
     );
+    const gasStarterFee = await this.getGasStarterFee(to, user);
 
```

Comment:
I'm not sure if we should include the gas starter fee in the payment infos, it may slow down the requests 🤔